changes/pr188.yaml

```diff
@@ -0,0 +1,8 @@
+enhancement:
+  - "Upgrade hasura version to 1.3.3 - [#188](https://github.com/PrefectHQ/server/pull/188)"
+If you are using managed database (e.g. Cloud SQL, etc) and upgrading hasura image version from 1.3.1 to 1.3.3, make sure that 
+hasura user is the owner of hdb_catalog schema. 
+You can find [more details in hasura documentation](https://hasura.io/docs/1.0/graphql/core/deployment/postgres-requirements.html#notes-for-managed-databases-aws-rds-gcp-cloud-sql-etc).
+Moreover, double check if hasura user owns hdb_catalog.insert_event_log function. 
+To check functions ownership, run following SQL query `SELECT proname, proowner::regrole FROM pg_proc WHERE pronamespace::regnamespace::text = 'hdb_catalog';`.
+To change function ownership, run `ALTER FUNCTION hdb_catalog.insert_event_log OWNER TO hasura;`.
```

Comment:
I'm not sure if this will end up where we want it. I'd put this in the helm chart docs in the "Troubleshooting" section under "Upgrading Hasura"

We may want to link to it from the changelog but I'm not sure.

@cicdw @jlowin interestingly, this is potentially a breaking change for the helm chart but not for other things, should we add a separate breaking change entry that just specifies that this may break a helm chart deployment with an external database?

Reply:
That seems reasonable and more visibility couldn't hurt, although I have a possibly ignorant question -- do people also use the helm chart for upgrading the infrastructure or is it a one-time use for setting it up?

Reply:
You can run [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/) and it's actually kind of smart about just redeploying things that have changed. I use it in development, I imagine some people would use it in their production environments 🤷